src/plugins/orangehrmPimPlugin/Service/PIMLeftMenuService.php

```diff
@@ -228,8 +224,6 @@ protected function generateMenuItems(?int $empNumber, bool $self): array
     /**
      * @param int|null $empNumber
      * @return bool
-     * @throws DaoException
-     * @throws ServiceException
```

Comment:
can remove?

Reply:
this also removed because of IDE suggestion